packages/cli/commands/project/create.js

```diff
@@ -45,8 +45,7 @@ exports.handler = async options => {
   logger.log('');
   logger.log(chalk.bold(i18n(`${i18nKey}.logs.welcomeMessage`)));
   uiFeatureHighlight([
-    'projectUploadCommand',
-    'projectDeployCommand',
```

Comment:
I removed these two because the dev command should probably be the first thing that a user does. But I can see the argument for keeping these in

Reply:
I think it makes sense to remove and put more focus on dev